apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/store.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import 'a8c-fse-common-data-stores';
```

Comment:
After a site is launched, the `editor-site-launch` module no longer loads.

The `wpcom-block-editor-nux` uses the site data store but it didn't import this `a8c-fse-common-data-stores` previously because it has always relied on `editor-site-launch` module to load it.

However, this `a8c-fse-common-data-stores`, also loads other unnecessary data store such is domains, plans, launch that is not relevant to `wpcom-block-editor-nux`. This is something to optimize for the future.

Reply:
Okay, scratch that. Changed my mind.
Now the `isGutenboarding` property either exists and hold the value "1" or does not exist at all.
The type definition is `isGutenboarding?: '1';`.
See https://github.com/Automattic/wp-calypso/pull/49004/commits/eeca2eb776c1b228f35958c3cc621dbe271278a4.

Reply:
I guess this last comment is related to https://github.com/Automattic/wp-calypso/pull/49004#discussion_r565238202 , right?

Reply:
LOL yes I replied in the wrong box.